WordPress/gutenberg
# Fix attribute escaping in Navigation block

## What?
Currently the Navigation block fails to use `esc_attr()` in some `aria-label` attributes.

## Why?
Good coding standards call for all html attributes to be escaped with `esc_attr()`. 

This is not a security issue, just a matter of code quality.

## How?
I considered late escaping as per #37880, but the way the code is written with the attribute name included in the variables suits an earlier `esc_attr__()` call.

There are some other items in the same `sprintf()` call in `render_block_core_navigation()` that might also need escaping, such as `$toggle_button_content` and `$toggle_close_button_content`. I haven't touched those as I'm unsure of the implications.

## Testing Instructions




1. View a page with a navigation block.
2. Confirm the `wp-block-navigation__responsive-container-open` has an `aria-label` attribute equivalent to `Open menu`.
3. Confirm generally that there is no change in behaviour to the Navigation block's rendered output.
